archiver.go

```diff
@@ -172,6 +177,34 @@ func (deb *ArchiveDeb) RelaxedUnpack(destination string) error {
 	return deb.Unpack(destination)
 }
 
+// Helper function for packing with external tool
+func pack(command []string) error {
+	return Command{}.Run("Pack", command...)
+}
+
+func (arc *ArchiveBase) Pack(destination string) error {
+	return fmt.Errorf("Pack is not supported for '%s'", arc.file)
+}
+
+func (tar *ArchiveTar) Pack(destination string) error {
+	command := []string{"tar"}
+	command = append(command, "-c")
+
+	if compression, ok := tar.options["tarcompression"]; ok {
```

Comment:
Somewhat orthogonally to your commit i really dislike the base archiver class having an options map that then gets abused with subclass specific strings.. It would be far nicer for the ArchiveTar struct to simply have fields for these things directly.  But not a blocker for this merge request :)